css-text/Overview.bs

```diff
@@ -1324,6 +1327,11 @@ Canonical order: N/A
       Shaping characters are still shaped as if the word were not
       broken, and grapheme clusters must stay together as one unit.
       No hyphenation character is inserted at the break point.
+    <dt><dfn>break-spaces</dfn></dt>
+    <dd>Line breaks are handled identically to ''overflow-wrap/normal'',
+      except that any sequence of <a>preserved</a> white space
+      that would otherwise overflow the line under these rules
+      must be broken after the last white space character that would fit the line.
```

Comment:
Very sorry for the slow answer.

> You want to break between spaces

correct, if `overflow-wrap` is `break-spaces.` you break and wrap in the middle of the space sequence instead of letting it overflow.

> but not before

Not before the first space  if `overflow-wrap` is only `break-spaces.` If it is `overflow-wrap:break-word | break-spaces`, you can also break before the first space, since `overflow-wrap: break-word` allows for breaks anywhere.

I think this is the best behavior, but I am not very strongly attached to this nuance, so if the rest of the proposal is OK and this is blocking, I am open to changing this part.

> or after?

I don't think there's anything in the wording I proposed that prohibits breaking after spaces. If the end of the line falls earlier than the last space, you should break at that earlier point, but if it falls right after that space, breaking there is the normal thing to do.

---
2 questions:
 - Are you OK with the behavior?
 - Do you think the proposed text (a) correctly and (b) understandably describes this behavior?